src/lambda_vm/gas.mli

```diff
@@ -0,0 +1,8 @@
+type t
+
+val make : initial_gas:int -> t
+
+val is_empty : t -> bool
+
+val burn_constant : t -> unit
```

Comment:
Can you add doc-strings for instance, in which case we use burn_constant, and burn_log2?